src/Compilers/CSharp/Portable/Symbols/Synthesized/SynthesizedAutoPropAccessorSymbol.cs

```diff
@@ -0,0 +1,205 @@
+                {
+                    return ImmutableArray<ParameterSymbol>.Empty;
+                }
+                return ImmutableArray.Create(SynthesizedParameterSymbol.Create(
+                    this,
+                    _property.TypeWithAnnotations,
+                    ordinal: 0,
+                    RefKind.None,
+                    name: ParameterSymbol.ValueParameterName));
+            }
+        }
+
+        public override ImmutableArray<MethodSymbol> ExplicitInterfaceImplementations => ImmutableArray<MethodSymbol>.Empty;
+
+        public override ImmutableArray<CustomModifier> RefCustomModifiers => _property.RefCustomModifiers;
+
+        public override Symbol AssociatedSymbol => _property;
+
+        public override Symbol ContainingSymbol => _property.ContainingSymbol;
+
+        public override ImmutableArray<Location> Locations => _property.Locations;
+
+        public override Accessibility DeclaredAccessibility => _property.DeclaredAccessibility;
+
+        public override bool IsStatic => _property.IsStatic;
+
+        public override bool IsVirtual => _property.IsVirtual;
+
+        public override bool IsOverride => _property.IsOverride;
+
+        public override bool IsAbstract => _property.IsAbstract;
+
+        public override bool IsSealed => _property.IsSealed;
+
+        public override bool IsExtern => _property.IsExtern;
+
+        public override ImmutableHashSet<string> ReturnNotNullIfParameterNotNull => ImmutableHashSet<string>.Empty;
+
+        internal override bool HasSpecialName => _property.HasSpecialName;
+
+        internal override MethodImplAttributes ImplementationAttributes => MethodImplAttributes.Managed;
+
+        internal override bool HasDeclarativeSecurity => false;
+
+        internal override MarshalPseudoCustomAttributeData? ReturnValueMarshallingInformation => null;
+
+        internal override bool RequiresSecurityObject => false;
+
+        internal override CallingConvention CallingConvention
+            => IsStatic ? CallingConvention.Default : CallingConvention.HasThis;
+
+        internal override bool GenerateDebugInfo => false;
+
+        public override DllImportData? GetDllImportData() => null;
+
+        internal override ImmutableArray<string> GetAppliedConditionalSymbols()
+            => ImmutableArray<string>.Empty;
+
+        internal override IEnumerable<SecurityAttribute> GetSecurityInformation()
+            => Array.Empty<SecurityAttribute>();
+
+        internal override bool IsMetadataNewSlot(bool ignoreInterfaceImplementationChanges = false) => IsVirtual && !IsOverride;
+
+        internal override bool IsMetadataVirtual(bool ignoreInterfaceImplementationChanges = false) => IsVirtual;
+
+        internal override bool SynthesizesLoweredBoundBody => true;
+
+        internal override void GenerateMethodBody(TypeCompilationState compilationState, DiagnosticBag diagnostics)
+        {
+            Debug.Assert(!(_property.BackingField is null));
```

Comment:
>Debug.Assert(!(_property.BackingField is null)); [](start = 12, length = 48)

If we are going after generalizable approach, do we still want to generate code for abstract case?